lighthouse-core/report/html/report-styles.css

```diff
@@ -98,13 +101,14 @@
   --color-red-700: #EB0F00;
   --color-red: #FF4E42;
   --color-white: #FFFFFF;
+  --env-name-min-width: 220px;
   --header-padding: 20px 0 20px 0;
+  --item-bg: var(--color-black-100);
+  --item-padding: 10px 0px;
```

Comment:
i presume `item` isn't specific to the env stuff?

Reply:
just checked, it's only uses for the env table. renamed.